lean-code-gate-v3/.agent/lean/lean_code_gate.py

```
@@ -2404,6 +2376,10 @@ def contract_errors(current_contract: dict[str, object], active_policy: dict[str
         for field, flag in required_fields:
             if not meaningful(current_contract.get(field)):
                 errors.append(f"Code work requires {flag}.")
+        proof_text = (" ".join(str(item) for item in proof_plan) if isinstance(proof_plan := current_contract.get("proof_plan"), list) else str(proof_plan or "")).lower()
+        tokens = tuple(token for token in verification_mode_tokens(active_policy) if token != "smoke-check" or not re.search(r"\b(pytest|unittest|nose|tox|jest|vitest|mocha|go test|cargo test|npm test|pnpm test|yarn test)\b", proof_text))
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "token"

(S105)